publications.html

```diff
@@ -51,7 +51,21 @@
     </nav>
 
     <!-- Content section -->
-
+    <section>
+      <div class="article container">
+        <a class="title">Recommending Tasks to Newcomers in OSS Projects: How Do Mentors Handle It?</a>
+        <span class="author">Sogol Balali, Umayal Annamalai, Susmita Padala, Bianca Trinkenreich, <a href="https://www.ime.usp.br/~gerosa/">Marco Aurelio Gerosa</a>, <a href="https://www.igor.pro.br/">Igor Steinmacher</a>, and <a href="https://epiclab.github.io/people.html">Anita Sarma</a></span>
+        <span class="venue">16th International Symposium on Open Collaboration (OpenSym), 2020.</span>
+      </div>
+    </section>
+      
+    <section>
+      <div class="article container">
+        <a class="title">Supporting Code Comprehension via Annotations: Right Information at the Right Time and Place</a>
+        <span class="author"><a href="https://epiclab.github.io/people.html">Umme Ayda Mannan</a>, <a href="https://www.ics.uci.edu/~iftekha/">Iftekhar Ahmed</a>, <a href="http://web.engr.oregonstate.edu/~jensenca/OSU_ENGR/index.html">Carlos Jensen</a>, and <a href="https://epiclab.github.io/people.html">Anita Sarma</a></span>
+        <span class="venue">ACM 28th SIGSOFT International Symposium on Foundations of Software Engineering (FSE), 2020.</span>
+      </div>
+    </section>
```

Comment:
The title, _Supporting Code Comprehension via Annotations: Right Information at the Right Time and Place_, is incorrect for this publication. It is a duplicate of the publication shown on [lines 70-76](https://github.com/EPICLab/EPICLab.github.io/blob/changes/publications.html#L70-L76).

Reply:
Looks like the publication should actually be titled: _On the Relationship Between Design Discussions and Design Quality: A Case Study of Apache Projects_ (per [Ayda's website](https://web.engr.oregonstate.edu/~mannanu/)).